docs/10-videre.adoc

```diff
@@ -3,21 +3,19 @@
 
 https://www.europeandataportal.eu/en/providing-data/goldbook[Open Data Goldbook for Data Managers and Data Holders]
 
-https://doc.difi.no/data/veileder-orden-i-eget-hus/[Veileder for orden i eget hus]
+https://www.digdir.no/informasjonsforvaltning/veileder-orden-i-eget-hus/2716[Veileder for orden i eget hus]
 
-https://doc.difi.no/data/veileder-for-beskrivelse-av-datasett/[Veileder for beskrivelse av datasett]
+https://data.norge.no/guide/veileder-beskrivelse-av-datasett[Veileder for beskrivelse av datasett]
 
-https://www.difi.no/rapport/2018/11/deling-av-data-konseptvalgutredning[Difi-rapport 2018:7 Deling av data - KVU]
 
 === Relevante standarder og spesifikasjoner
 
-https://doc.difi.no/dcat-ap-no/[Standard for beskrivelse av datasett og datakataloger]
+https://data.norge.no/specification/dcat-ap-no[Standard for beskrivelse av datasett og datakataloger]
```

Comment:
Tittelen har blitt endret i mellomtiden: Standard for beskrivelse av datasett, datatjenester og datakataloger (DCAT-AP-NO)